Makefile

```diff
@@ -84,6 +85,7 @@ build-coverage: clean $(CMDS)
 
 build-linux: build_cmd=build
 build-linux: arch_flags=GOOS=linux GOARCH=386
+build-linux: BUILD_TAGS="json1 experimental_metrics"
```

Comment:
Any idea on why we need to update the build-linux target as well? At a glance locally, it doesn't look like this is used anywhere to drive CI.

Reply:
I believe the e2e test relies on the [e2e-kind GitHub Workflow](https://github.com/operator-framework/operator-lifecycle-manager/blob/6dc902654d4f2089da9da56cc6b1d574be995d20/.github/workflows/e2e-kind.yml), which calls the`run-local` target [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/.github/workflows/e2e-kind.yml#L26), which calls the build-linux target [here](https://github.com/operator-framework/operator-lifecycle-manager/blob/41d0002d659f43eb07d45609dc9b56478fbc3c50/Makefile#L284).

This change might be better placed in the `run-local` target, thoughts?